db/fixtures/matches.rb

```diff
@@ -1,9 +1,11 @@
-students = [*1..20].shuffle
```

Comment:
You'll need to name this file appropriately in order to put it alphabetically _after_ `students.rb` and `tutors.rb`. Otherwise, any pg database will complain when trying to seed about the `id`s not existing.